### PR TITLE
Make swaybar and swaybg shell surfaces

### PR DESCRIPTION
**Pull request by deklov**
Per discussion in #343

### CODE REVIEW

**Comment by deklov** [comment]
Just realized I used spaces instead of tabs. I'll fix that once I got feedback.

File: wayland/window.c [hunk 1]
```diff
@@ -59,6 +59,13 @@ static const struct wl_shell_surface_listener surface_listener = {
 	.configure = shell_surface_configure
 };
 
+void window_make_shell(struct window *window)
+{
+        window->shell_surface = wl_shell_get_shell_surface(window->registry->shell, window->surface);
+        wl_shell_surface_add_listener(window->shell_surface, &surface_listener, window);
+        wl_shell_surface_set_toplevel(window->shell_surface);
+}
```
Comment (SirCmpwn) [inline wayland/window.c]: Can't we use wlc_view_from_surface?

Comment (deklov) [inline wayland/window.c]: This function is called on the client side.

Comment (SirCmpwn) [inline wayland/window.c]: Oh, my bad.

**Comment by deklov** [comment]
Other than changing space to tabs. Is there anything else you want me to change?

**Pull request closed without merging**